frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBindings.tsx

```diff
@@ -60,6 +60,7 @@ export const EditDataModelBindings = ({
 
   return (
     <div key={uniqueKey || ''}>
+      <p style={{ border: '1px solid red' }}>HALLAAAA</p>
```

Comment:
Halla! What is this? 😆

Reply:
Hahahahahahha, not supposed to be there 😆 Thanks!